src/internal/components/abstract-switch/index.tsx

```diff
@@ -74,18 +80,33 @@ export default function AbstractSwitch({
   }
 
   return (
-    <span {...rest} className={clsx(styles.wrapper, rest.className)} ref={__internalRootRef}>
+    <span
+      {...rest}
+      className={clsx(styles.wrapper, rest.className)}
+      ref={__internalRootRef}
+      {...getAnalyticsLabelAttribute(label ? `.${analyticsSelectors.label}` : `.${analyticsSelectors['native-input']}`)}
```

Comment:
Why do we chose either a label or aria-label and not both? Also, if both are defined the ARIA is usually more descriptive.